packages/daimo-api/src/router.ts

```
@@ -48,12 +49,23 @@ export function createRouter(
     return result;
   });
 
-  const publicProcedure = trpcT.procedure.use(tracerMiddleware);
+  const corsMiddleware = trpcT.middleware(async (opts) => {
+    opts.ctx.res.setHeader("Access-Control-Allow-Origin", "*");
+    return opts.next();
+  });
+
+  const publicProcedure = trpcT.procedure
+    .use(corsMiddleware)
+    .use(tracerMiddleware);
 
   return trpcT.router({
     health: publicProcedure.query(async (_opts) => {
-      return "healthy";
+      // Push Notifier is the last service to load.
+      const ret = notifier.isInitialized ? "healthy" : "pending";
+      console.log(`[API] health check: ${ret}`);
+      return ret;
```

i think the pending still returns 200OK, you should throw an error for #388